### PR TITLE
Feature/org rollup 2

### PR DESCRIPTION
**Pull request by agduncan94**
https://github.com/ga4gh/dockstore/issues/2165
https://github.com/ga4gh/dockstore/issues/2185
* Use display names for add to collection
* Are you sure dialog for removing members of orgs

### CODE REVIEW

File: src/app/organizations/organization-members/delete-username-confirm-dialog.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<h1 mat-dialog-title>Remove user from organization</h1>
+
+<div mat-dialog-content>
+  <p>Are you sure you want to <strong>remove</strong> the user <strong>{{data.role.user.username}}</strong> from the organization <strong>{{data.role.organization.displayName}}</strong>?</p>
+</div>
+<div mat-dialog-actions>
+  <div fxFlex></div>
```
Comment (garyluu) [inline src/app/organizations/organization-members/delete-username-confirm-dialog.html]: Empty div: 
- https://stackoverflow.com/a/38638044/4484759
- https://stackoverflow.com/a/14182485/4484759

File: src/app/organizations/organization-members/organization-members.component.ts [hunk 2]
```diff
@@ -76,23 +96,30 @@ export class OrganizationMembersComponent implements OnInit {
...
    */
-  addUser() {
-    this.alertService.clearEverything();
-    this.matDialog.open(UpsertOrganizationMemberComponent,
-      { data: { mode: TagEditorMode.Add, username: null, role: null }, width: '600px' });
+  removeUserDialog(organizationUser: OrganizationUser) {
+    const dialogRef = this.matDialog.open(OrganizationMemberRemoveConfirmDialogComponent,
+      { data: { role: organizationUser }, width: '600px' });
+
+    dialogRef.afterClosed().subscribe(result => {
```
Comment (garyluu) [inline src/app/organizations/organization-members/organization-members.component.ts]: check if need to unsubscribe

File: src/app/organizations/organization-members/organization-members.component.ts [hunk 3]
```diff
@@ -27,6 +27,26 @@ import { OrganizationMembersService } from '../state/organization-members.servic
...
+  templateUrl: 'delete-username-confirm-dialog.html',
+})
+export class OrganizationMemberRemoveConfirmDialogComponent {
+
+  constructor(
+    public dialogRef: MatDialogRef<OrganizationMemberRemoveConfirmDialogComponent>,
+    @Inject(MAT_DIALOG_DATA) public data: DialogData) {}
+
+  onNoClick(): void {
+    this.dialogRef.close();
```
Comment (garyluu) [inline src/app/organizations/organization-members/organization-members.component.ts]: there's directive for this

File: src/app/organizations/organization-members/delete-username-confirm-dialog.html [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<h1 mat-dialog-title>Remove user from organization</h1>
+
+<div mat-dialog-content>
+  <p>Are you sure you want to <strong>remove</strong> the user <strong>{{data.role.user.username}}</strong> from the organization <strong>{{data.role.organization.displayName}}</strong>?</p>
+</div>
+<div mat-dialog-actions fxLayoutAlign="flex-end center">
```
Comment (garyluu) [inline src/app/organizations/organization-members/delete-username-confirm-dialog.html]: just copy the style guide or update the style guide with something better

File: src/app/organizations/organization-members/organization-members.component.ts [hunk 5]
```diff
@@ -27,6 +27,21 @@ import { OrganizationMembersService } from '../state/organization-members.servic
...
 import { UpsertOrganizationMemberComponent } from '../upsert-organization-member/upsert-organization-member.component';
 
+@Component({
+  selector: 'organization-member-remove-dialog',
+  templateUrl: 'delete-username-confirm-dialog.html',
+})
+export class OrganizationMemberRemoveConfirmDialogComponent {
+
+  constructor(
+    public dialogRef: MatDialogRef<OrganizationMemberRemoveConfirmDialogComponent>,
```
Comment (garyluu) [inline src/app/organizations/organization-members/organization-members.component.ts]: dialogRef is unused i think

File: src/app/organizations/organization-members/delete-username-confirm-dialog.html [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+  <button mat-button mat-dialog-close>No Thanks</button>
```
Comment (garyluu) [inline src/app/organizations/organization-members/delete-username-confirm-dialog.html]: check that this is the focus

Comment (agduncan94) [reply src/app/organizations/organization-members/delete-username-confirm-dialog.html]: It is

**Pull request merged**